extract/news/yna_crawler.py

```
@@ -0,0 +1,125 @@
+from datetime import datetime
+from typing import Optional, List
+import re
+import json
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+import boto3
+import pyarrow as pa
+import pyarrow.parquet as pq
+from conf import *
```

Ruff (0.8.2)

`from conf import *` used; unable to detect undefined names

(F403)